app/src/main/res/xml/remote_config_defaults.xml

```diff
@@ -450,8 +450,7 @@
     </entry>
     <entry>
         <key>v2_encounterWarning</key>
-        <value>Setkali jste se s osobou, u které byla prokázána nákaza COVID-19. Sledujte svůj
-            zdravotní stav.
+        <value>Setkali jste se s osobou, u které bylo potvrzeno onemocnění COVID-19. Sledujte svůj zdravotní stav.
```

Comment:
Předělal jsem to všude na koronavirus viz Slack. Až bude překlad tak plánuju updatnout RC i defaulty.